lib/main.dart

```diff
@@ -100,8 +121,11 @@ class _FullDogViewState extends State<FullDogView> {
       appBar: AppBar(
         title: Row(
           children: <Widget>[
-            const Text('Dog Stats'),
             const Icon(FontAwesomeIcons.bone),
+            // TODO: does this need to be const?
+            // TODO: use some kind of box instead of Padding Widget
+            const Padding(padding: EdgeInsets.only(left: 20.0)),
```

Comment:
regarding your TODO, yes it can be. technically nothing *needs* to be const, it just improves performance. I can make those changes.